docs/env_vars_and_config.md

```diff
@@ -1,69 +1,150 @@
 
+#### Example #1 - Setting ENV vars upon login by exporting them using a user-initialization script
+```bash
+# Export these vars directly from within your preferred user initialization script (.bash_profile, .bashrc, .profile) and 
+# they will be set and ready to be used upon login to the terminal and throughout the entirety of your user session
+
+export APP_ENV=prod
+export ILIOS_MAILER_URL=null
+export ILIOS_AUTHENTICATION_TYPE=form
+export ILIOS_FILE_SYSTEM_STORAGE_PATH=/var/www/files/learning_materials
+export ILIOS_DATABASE_URL=mysql://ilios_user:ili0s_passw0rd@database.example.com/ilios_db
+export ILIOS_DATABASE_MYSQL_VERSION=5.7
+export ILIOS_LOCALE=en
+export ILIOS_SECRET=ThisTokenIsNotSoSecretChangeIt
 ```
-ILIOS_DATABASE_URL
-ILIOS_DATABASE_MYSQL_VERSION
-ILIOS_MAILER_TRANSPORT
-ILIOS_MAILER_HOST
-ILIOS_MAILER_USER
-ILIOS_MAILER_PASSWORD
-ILIOS_LOCALE
-ILIOS_SECRET
+Doing it this way ensures that these ENV vars will always be available to your user while you work and/or run commands directly from the command line.
+
+#### Example #2 - Setting ENV vars at the top of a shell script that runs the desired console command
+
+Let's say that you are going to run the `/bin/setup` command to install a completely new version of Ilios.  For this approach, you would create a shell script that runs `bin/setup` as its final step, and set the ENV vars above it in the file, like so:
+```bash
+#!/bin/bash
+
+# set the environment variables
+ILIOS_APP_DIR=/var/www/ilios
+APP_ENV=prod
+ILIOS_MAILER_URL=null
+ILIOS_AUTHENTICATION_TYPE=form
+ILIOS_FILE_SYSTEM_STORAGE_PATH=/var/www/files/learning_materials
+ILIOS_DATABASE_URL=mysql://ilios_user:ili0s_passw0rd@database.example.com/ilios_db
+ILIOS_DATABASE_MYSQL_VERSION=5.7
+ILIOS_LOCALE=en
+ILIOS_SECRET=ThisTokenIsNotSoSecretChangeIt
+
+#run the setup command
+$ILIOS_APP_DIR/bin/setup
 ```
 
-The user running the web server processes on your system will most likely NOT have an environment with a typical login shell like `BASH` or `csh`, so setting the ENV vars cannot be done with the usual method (eg, running `export ILIOS_SECRET='SomethingSecret'` at the command line). In these cases, the web service user's ENV variable values need to be set in the web server configuration directly (eg, within the `httpd.conf` file on Apache).
+Note that doing it this way will only set the ENV variables for the duration of the execution of the `bin/setup` command and you will need to create/update a script like this for each command-line command you want to run.
 
-To verify which of these ENV vars, if any, are set for your web service user, create a php file on your webserver with the following content and then view the file in your browser:
+#### Example #3 - Setting ENV vars at the command line directly
 
-```php
-<?php
-phpinfo();
+Technically, you can set all of the ENV vars at the command line on the same line and at the same time that you run the `bin/setup` command, like so:
```

Comment:
I would change this to "While possible to set all of the ENV vars at the command line.....when you run `bin/setup`, we do NOT recommend this approach. However, if you choose to do so, the steps would be...." or something similar. Want to dissuade right out of the gate.